app/src/main/java/com/kickstarter/viewmodels/projectpage/CrowdfundCheckoutViewModel.kt

```diff
@@ -158,12 +157,11 @@ class CrowdfundCheckoutViewModel(val environment: Environment, bundle: Bundle? =
                 sharedPreferences
             )
 
-            when (flowContext) {
-                PledgeFlowContext.NEW_PLEDGE,
-                PledgeFlowContext.CHANGE_REWARD -> getPledgeInfoFrom(pData)
-                PledgeFlowContext.MANAGE_REWARD,
```

Comment:
"Manage_Reward" is used for both updating selected reward and changing reward, it describes the fact of having to present to the user RewardsCarousel not granular enough. 
Exchanged to use `PledgeReason` instead of `PledgeFlowContext` as it's uniquely identified.